webpack/assets/javascripts/react_app/routes/common/EmptyPage/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../../common/I18n';
+import DefaultEmptyState from '../../../components/common/EmptyState';
+import './emptypage.scss';
+
+const EmptyPage = ({ type, text, name }) => (
```

Comment:
whats the difference between empty page and empty state?

Reply:
no difference, just a default one with i18n

Reply:
can't we add the i18n? I think we already have a wrapper in foreman for it?